src/test/unit/util.hpp

```diff
@@ -89,6 +89,39 @@ void match_csv_columns(const Eigen::MatrixXd& samples,
   }
 #endif
 
+/**
+ * Gets the path separator for the OS.
+ *
+ * @return '\' for Windows, '/' otherwise.
+ */
+char get_slash() {
+#if defined(WIN32) || defined(_WIN32) \
+    || defined(__WIN32) && !defined(__CYGWIN__)
+  static char path_separator = '\\';
+#else
+  static char path_separator = '/';
+#endif
+  return path_separator;
+}
+
+/**
+ * Returns the path as a string with the appropriate path separator.
+ *
+ * @param file_path vector of strings representing path to the file
+ * @return the string representation of the path
+ */
+std::string paths_to_fname(const std::vector<std::string>& path) {
```

Comment:
Are these intentionally duplicates of the ones in `test/unit/io/json/util`?

Reply:
added twice by mistake - leaving here, removing from `test/unit/io/json/util`.